excercsises/2_page_object/pages/home.js

```diff
@@ -9,13 +9,20 @@ var HomePage = function() {
   this.getTitle = function() {
     return browser.driver.getTitle();
   };
-  this.getItemsFromContactPage = ()=>{
-    return element(by.xpath('//*/span/h3')),element(by.xpath('//*/fieldset/h3'));
-  }
-  this.getContactText = () => {
+  this.findAdressAndgetInTouch=  async ()=>{
+    let array = [];
+    element(by.xpath('//fieldset/h3')).getText().then((function(el) {
+      array.push(el);
+  }))
+  element(by.xpath('//span[2]/h3')).getText().then((function(el) {
+      array.push(el);
+  }))
+    return array;
+    }
+  this.findContactBtnText = () => {
     return element(by.xpath("//a[contains(text(), 'Contact')]")).getText();
   };
-  this.getContactClick = () => {
+  this.findContactBtn = () => {
```

Comment:
The main action of this method is not finding button. The 'element by' does it but you are clicking the button here to do something. It should be more like openContact(), sometimes it is even good to create a new class for e.g. navbar or menu as all these buttons on the top of the page are inside 'navbar-inner' element, but lets skip that part here and stay with just home page.